lib/puppet/type/iis_app_pool.rb

```diff
@@ -0,0 +1,12 @@
+require 'puppet/parameter/boolean'
+require_relative '../../puppet_x/puppetlabs/iis/property/string'
+
+Puppet::Type.newtype(:iis_app_pool) do
```

Comment:
This is part of why manifests are good design artifacts. Generally the puppet idiom is to have more underscores. So the question is `iis_application_pool` vs `iis_app_pool`. Which one it is, I have no preference. I'll defer to the domain expert on that one.

In the end it is something a script can fix for us in the last moment.